### PR TITLE
fixed introspection for schema description and specifiedByURL

### PR DESCRIPTION
**Pull request by Code-Hex**
fixed introspection to support the latest spec.

https://spec.graphql.org/October2021/#sec-Schema-Introspection

ref: https://github.com/vektah/gqlparser/pull/182

This PR can avoid like this problem: https://github.com/99designs/gqlgen/issues/1636

Describe your PR and link to any relevant issues. 

I have:
 - [ ] Added tests covering the bug / feature (see [testing](https://github.com/99designs/gqlgen/blob/master/TESTING.md))
 - [ ] Updated any relevant documentation (see [docs](https://github.com/99designs/gqlgen/tree/master/docs/content))

### CODE REVIEW

File: graphql/introspection/schema.go [hunk 1]
```diff
@@ -11,6 +11,11 @@ type Schema struct {
 	schema *ast.Schema
 }
 
+func (s *Schema) Description() string {
+	// TODO(codehex): implement description in gqlparser/ast.Schema
+	return ""
+}
```
Comment (Code-Hex) [inline graphql/introspection/schema.go:17]: required: https://github.com/vektah/gqlparser/pull/183

**Comment by Code-Hex** [comment]
I've checked with rewrote go.mod like below

```
replace github.com/vektah/gqlparser/v2 => ../gqlparser
```

File: TESTING.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ in another terminal
 ```bash
 cd integration
 npm install
-SERVER_URL=http://localhost:8080/query ./node_modules/.bin/graphql get-schema
+./node_modules/.bin/graphql-codegen
```
Comment (Code-Hex) [inline TESTING.md:34]: use graphql-codegen because need to upgrade graphql-js to v16.3.0 (the same as gqlparser) and graphql-cli is wrapper for it

**Comment by Code-Hex** [comment]
@StevenACoffman Could you check?

File: go.mod [hunk 3]
```diff
@@ -13,8 +13,8 @@ require (
 	github.com/mitchellh/mapstructure v1.2.3
 	github.com/stretchr/testify v1.4.0
 	github.com/urfave/cli/v2 v2.3.0
-	github.com/vektah/gqlparser/v2 v2.3.1
-	golang.org/x/tools v0.1.5
+	github.com/vektah/gqlparser/v2 v2.3.2-0.20220228141359-a1885932a1bc
```
Comment (Code-Hex) [inline go.mod:16]: use the latest

**Review by StevenACoffman — APPROVED** [review]
Thanks!

File: integration/schema-expected.graphql [hunk 4]
```diff
@@ -32,28 +29,29 @@ enum ErrorType {
 
 input ListCoercion {
   enumVal: [ErrorType]
-  strVal: [String]
   intVal: [Int]
   scalarVal: [Map]
+  strVal: [String]
 }
 
+""""""
```
Comment (giautm) [inline integration/schema-expected.graphql:37]: Why's this happened?

Comment (Code-Hex) [reply integration/schema-expected.graphql:37]: check https://github.com/99designs/gqlgen/pull/2008

**Pull request merged**